Code/Tools/SceneAPI/SceneData/Tests/SceneManifest/SceneManifestRuleTests.cpp

```diff
@@ -127,9 +175,74 @@ namespace AZ
                 UnitTest::AllocatorsFixture::TearDown();
             }
 
+            void PrepareMockPythonInterface()
+            {
+                ON_CALL(m_data->m_editorPythonEventsInterface, StartPython(::testing::_))
+                    .WillByDefault([](bool) -> bool
+                    {
+                        return true;
+                    });
+
+                ON_CALL(m_data->m_editorPythonEventsInterface, StopPython(::testing::_))
+                    .WillByDefault([](bool) -> bool
+                    {
+                        return true;
+                    });
+
+                ON_CALL(m_data->m_editorPythonEventsInterface, IsPythonActive())
+                    .WillByDefault([]() -> bool
+                    {
+                        return true;
+                    });
```

Comment:
I think these can be simplified by using: `WillByDefault(Return(true))`

Reply:
nice call, will do